### ISSUE TITLE
Accessibility: JAWS, manual, Chrome extension checking

### ISSUE THREAD

**Issue by chirp1** [OpenLiberty/docs#5827]
Generally, the documentation needs to be tested and accessibility violations fixed. At least some would likely need to be fixed by the open liberty site team.
JAWS testing of just a few topics pointed out issues:

For code snippets, a copy button that JAWS can read needs to be on each code snippet. 
- Instead of a button, it is a graphic. Make it a button.
- JAWS can read the graphic only on the first code snippet, not on the ones that follow.
The problem can be seen in the following two topics.
   - Log and trace configuration: https://openliberty.io/docs/latest/log-trace-configuration.html?_ga=2.138938227.1100166455.1663769468-1172611259.1588265214#rollover

   - HTTP Access logging: https://openliberty.io/docs/latest/access-logging.html?_ga=2.138938227.1100166455.1663769468-1172611259.1588265214#_time_based_http_access_log_rollover
     - Another problem with this topic is that "Settings for a common log" and "Settings for distinct logs for each endpoint" read more like subheadings, but they are coded as list items.
     
     Using the Accessibility Checker Chrome extension in the OpenLiberty.io framework, violations with color contrast, possibly with the version selector, problems tabable elements.

**Comment by dmuelle** [comment]
opened https://github.com/OpenLiberty/openliberty.io/issues/2885 for the code block button problem; I encountered this too

**Comment by dmuelle** [comment]
fixed the heading issue, published with 22.0.0.12
